docs/architecture.md

```
@@ -0,0 +1,160 @@
+# アーキテクチャ
+
+## アセンブリ構成
+
+avatar-vcs は UnityEditor 依存を持つコードと持たないコードを明確に分離しています。
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)